### ISSUE TITLE
Allow the user to have the notification open several seconds after the video has ended. (new option in settings)

### ISSUE THREAD

**Issue by anjandev** [Desuvader/Hummingbird-Updater#1]
I hate how it shows the notification as soon as you start the video. I have this habit of fullscreening the video as soon as I start the video and I always miss the notification.

**Comment by tofuness** [comment]
I see. The scrobbler currently doesn't have a way to detect how much you've watched of an episode, simply because it doesn't actually hook into the process.

I think there might be a way to make the notification _always_ stay on top. Would that be good?

**Comment by anjandev** [comment]
Yes! The program could have an option to always show the notification unless dismissed.
